docs/reference/transform/apis/get-transform-stats.asciidoc

```diff
@@ -262,7 +270,7 @@ The following example gets usage information for the `ecommerce_transform`
 
 [source,console]
 --------------------------------------------------
-GET _transform/ecommerce_transform/_stats
+GET _transform/ecommerce-customer-transform/_stats
```

Comment:
look like line 268 has to be changed, too:

> The following example gets usage information for the `ecommerce_transform`

Reply:
Thanks!  I've removed that qualifier